agagd/templates/agagd_core/member.html

```diff
@@ -1,18 +1,20 @@
 
-{% extends "base.html" %} 
-{% load staticfiles %} 
+{% extends "base.html" %}
+{% load staticfiles %}
 {% load render_table from django_tables2 %}
 
-{% block title %} Player card for {{player.full_name}} {% endblock %} 
+{% block title %} Player card for {{player.full_name}} {% endblock %}
 {% block stylesheets %} {{ block.super }} {% endblock %}
-{% block header %} 
+{% block header %}
 {% endblock %}
 
-{% block content %} 
+{% block content %}
 {{ block.super }} 
-<h2> Player card for <b>{{player.full_name}}</b>, ID #{{player.member_id}}.</h2> 
+<h2> Player card for <b>{{player.full_name}}</b>, ID #{{player.member_id}}.</h2>
 {% include "agagd_core/ratings_graph.html" %}
 <p> AGA member since: {{ player.join_date }} <p>
+<p style="font-size:75%;">Expiration Date: {{ player.renewal_due}} </p>
+<p style="font-size:75%;">Membership type: {{ player.type}} </p>
```

Comment:
In order to access `player.renewal_due` and `player.type` they need to be part of the `Member` model in [models.py](../agagd_core/models.py).

`renewal_date` and `type` should have a space after i.e., `{{ player.renewal_due }}`.

Better if these styles were manage in [agagd.css](../static/css/agagd.css) instead of with an inline style. Could you add a class to manage the styles?